lib/src/presentations/view_company_profile/screen/view_company_profile_view.dart

```diff
@@ -0,0 +1,295 @@
+              headerSliverBuilder: (_, __) => [
+                _buildAppBar(),
+                SliverToBoxAdapter(
+                  child: Container(
+                    padding: const EdgeInsets.symmetric(
+                        horizontal: 20, vertical: 18),
+                    child: _buildButtonProfile(),
+                  ),
+                ),
+                SliverAppBar(
+                  backgroundColor: AppColors.scaffoldBackground,
+                  pinned: true,
+                  centerTitle: true,
+                  automaticallyImplyLeading: false,
+                  scrolledUnderElevation: 0,
+                  expandedHeight: 0,
+                  toolbarHeight: 60,
+                  title: _buildTabBar(onTap: tabsRouter.setActiveIndex),
+                ),
+              ],
+              body: child,
+            ),
+          ),
+        );
+      },
+    );
+  }
+
+  Widget _buildAppBar() {
+    return BlocBuilder<ViewCompanyProfileCubit, ViewCompanyProfileState>(
+      builder: (context, state) {
+        return SliverAppBar(
+          actions: [
+            IconButton(
+              onPressed: () {},
+              icon: const Icon(FontAwesomeIcons.ellipsisVertical),
+            ),
+            const SizedBox(width: 5),
+          ],
+          leading: IconButton(
+            onPressed: context.router.pop,
+            icon: const Icon(Icons.arrow_back_rounded, color: Colors.black),
+          ),
+          backgroundColor: state.isTop ? Colors.white : Colors.transparent,
+          elevation: 0,
+          pinned: true,
+          centerTitle: true,
+          stretch: true,
+          expandedHeight: 240,
+          scrolledUnderElevation: 0,
+          title: AnimatedOpacity(
+            opacity: !state.isTop ? 0.0 : 1.0,
+            duration: const Duration(milliseconds: 300),
+            child: Text(
+              state.user?.name ?? "",
+              style: const TextStyle(fontWeight: FontWeight.w700),
+            ),
+          ),
+          flexibleSpace: FlexibleSpaceBar(
+            stretchModes: const [StretchMode.zoomBackground],
+            expandedTitleScale: 1.0,
+            centerTitle: false,
+            titlePadding: const EdgeInsets.all(16),
+            background: _buildBackgroundAppbar(context, state: state),
+          ),
+        );
+      },
+    );
+  }
+
+  Widget _buildBackgroundAppbar(
+    BuildContext context, {
+    required ViewCompanyProfileState state,
+  }) {
+    return Stack(
+      alignment: Alignment.center,
+      children: [
+        Column(
+          children: [
+            Container(height: 120, color: Colors.white),
+            Container(
+              height: 120,
+              width: double.infinity,
+              decoration: const BoxDecoration(color: Color(0xFFF3F2F2)),
+              child: Column(
+                children: [
+                  const SizedBox(height: 28),
+                  state.user != null
```

Comment:
Should be put it into Widget function. It's so clearly

Reply:
I fix it